web/pgadmin/static/js/components/ReactCodeMirror/CustomEditorView.js

```diff
@@ -30,11 +30,13 @@ export default class CustomEditorView extends EditorView {
     this._cleanDoc = this.state.doc;
   }
 
-  getValue(tillCursor=false) {
+  getValue(tillCursor=false, lineSep=false) {
```

Comment:
`lineSep` should be string here, but default value is boolean.

Reply:
my bad.